project/Dependencies.scala

```diff
@@ -5,7 +5,6 @@ object Dependencies {
   val akkaHttpV = "10.6.3"
   val jacksonV = "2.18.3"
   val jacksonHotfixV = "2.18.3" // for when only some of the Jackson libs have hotfix releases
-  val nettyV = "4.1.119.Final"
```

Comment:
all of the libraries removed in this PR are no longer needed, given all the code deletion.